targets/mbedos5/tools/js2c.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python
```

Comment:
I'm have the same view as @akosthekiss, the pin js file generation and extraction should be done in a different script and not in the `js2c.py`.